daemon/plugin-fuzz/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
+name = "ethersync-plugin-fuzz"
+description = "Fuzzing the daemon via simulated plugin input over the socket."
+version = "0.0.0"
+edition = "2021"
+publish = false
+
+[[test]]
+name = "random-input-nvim"
```

Comment:
I'd favor keeping the name "fuzzer" for this one, as we refer to that component like that, anyway :) And because it tests the whole chain Vim <-> Daemon <-> Daemon <-> Vim.

Reply:
Thanks that makes perfect sense! Fixed.